Dockerfile

```diff
@@ -27,9 +27,9 @@ ENV TARGET_OS=${TARGETOS}
 
 RUN make manager
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.8
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.2
```

Comment:
@igor-karpukhin are you okay with this update?
It would not affect openshift certification process, right?

Reply:
@helderjs if it passed the upgrade test, then that's okay. We can always rebuild it with the previous one if it won't pass certification.